app/src/sharedTest/java/org/oppia/app/profile/AdminPinActivityTest.kt

```diff
@@ -110,6 +112,86 @@ class AdminPinActivityTest {
     }
   }
 
+  @Test
+  fun testAdminPinActivity_rotateScreen_inputPin_inputConfirmPin_clickSubmit_checkOpensAddProfileActivity() {
```

Comment:
Replace `rotateScreen` with `configurationChange` in all test cases.
Just to be clear `rotateScreen` is also correct but in all test cases we are using `configurationChange` so its better if we change it here too so that we can keep it consistent across all files.